dlc/src/main/scala/org/bitcoins/dlc/builder/DLCTxBuilder.scala

```diff
@@ -218,33 +163,12 @@ case class DLCTxBuilder(
 
     builderF.flatMap(_.buildRefundTx())
   }
-
-  /** Constructs the unsigned mutual close transaction for a given oracle signature */
-  def buildMutualCloseTx(sig: SchnorrDigitalSignature): Future[Transaction] = {
-    val builderF = for {
-      fundingTx <- buildFundingTx
-    } yield {
-      val fundingOutPoint = TransactionOutPoint(fundingTx.txId, UInt32.zero)
-
-      DLCMutualCloseTxBuilder(offer.oracleInfo,
-                              offerFinalAddress.scriptPubKey,
-                              offerOutcomes,
-                              acceptFinalAddress.scriptPubKey,
-                              acceptOutcomes,
-                              fundingOutPoint)
-    }
-
-    builderF.flatMap(_.buildMutualCloseTx(sig))
-  }
 }
 
 object DLCTxBuilder {
 
-  /** Experimental approx. vbytes for a CET */
-  val approxCETVBytes = 190
-
-  /** Experimental approx. vbytes for a closing tx spending ToLocalOutput */
-  val approxToLocalClosingVBytes = 122
+  /** Place-holder vbytes for a CET/Refund Tx TODO: compute this 165? */
+  val approxClosingVBytes: Long = 200L
```

Comment:
From running the DLC execution tests I am getting a 169 vSize for 2 output ones and 138 for single output